spec/patreon_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe ::Patreon do
+
+  let(:user1) { Fabricate(:user) }
+  let(:user2) { Fabricate(:user) }
+  let(:group) { Fabricate(:group) }
+  let(:filters) { { group.id.to_s => [] } }
+
+  before {
```

Comment:
In specs, we mostly use `do ... end` block notation and only use `{ ... }` notations for single lines.